lib/cdo/rack/upgrade_insecure_requests.rb

```diff
@@ -48,10 +48,10 @@ def call(env)
             "child-src blob: ",
             "script-src 'self' https: 'unsafe-inline' https://vaas.acapela-group.com 'unsafe-eval'",
             "style-src 'self' https: 'unsafe-inline'",
-            "img-src 'self' https: data: blob:",
+            "img-src 'self' https: data: blob: http://javabuilderbeta-output.dev-code.org",
             "font-src 'self' https: data:",
             "connect-src 'self' https: https://api.pusherapp.com wss://ws.pusherapp.com wss://*.firebaseio.com http://localhost:8080 https://curriculum.code.org/ wss://*.code.org wss://javabuilderbeta.dev-code.org",
-            "media-src 'self' https: http://vaas.acapela-group.com",
+            "media-src 'self' https: data: http://vaas.acapela-group.com http://javabuilderbeta-output.dev-code.org",
```

Comment:
The Javabuilder output file web server supports SSL/TLS. Can we switch to: `https://javabuilderbeta-output.dev-code.org` ?

Reply:
Done!

Reply:
What does `data:` do here? Are we sure that will work with acapela-group?

Reply:
`data:` is included to allow base64 encoded strings as sources (`img-src` has this as well) - it shouldn't affect acapela-group as I understand since it's a separate entry (I believe this is why base64 encoded gifs were appearing on staging/production, but audio was not)

Reply:
Ahh ok. Makes sense.